src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -265,16 +271,24 @@ private constructor(
     ): Launcher<CodyAgentServer> {
       return Launcher.Builder<CodyAgentServer>()
           .configureGson { gsonBuilder ->
-            gsonBuilder
-                // emit `null` instead of leaving fields undefined because Cody
-                // VSC has many `=== null` checks that return false for undefined fields.
-                .serializeNulls()
-                .registerTypeAdapter(CompletionItemID::class.java, CompletionItemIDSerializer)
-                .registerTypeAdapter(ContextItem::class.java, ContextItem.deserializer)
-                .registerTypeAdapter(Speaker::class.java, speakerDeserializer)
-                .registerTypeAdapter(Speaker::class.java, speakerSerializer)
-                .registerTypeAdapter(URI::class.java, uriDeserializer)
-                .registerTypeAdapter(URI::class.java, uriSerializer)
+            run {
+              gsonBuilder
+                  // emit `null` instead of leaving fields undefined because Cody
+                  // VSC has many `=== null` checks that return false for undefined fields.
+                  .serializeNulls()
+                  .registerTypeAdapter(ContextItem::class.java, ContextItem.deserializer)
```

Comment:
Is there any way to register all classes from this directory/package, so we never miss one?